pulsar-client/src/main/java/org/apache/pulsar/client/impl/HttpLookupService.java

```diff
@@ -88,6 +92,29 @@ public String getServiceUrl() {
     	return httpClient.url.toString();
     }
 
+    @Override
+    public CompletableFuture<List<String>> getTopicsUnderNamespace(NamespaceName namespace) {
+        CompletableFuture<List<String>> future = new CompletableFuture<>();
+        httpClient
+            .get(String.format("admin/namespaces/%s/destinations", namespace), String[].class)
```

Comment:
We should be able to deserialize directly into a `List<String>` with Jackson, by passing `List.class`.

Reply:
Thanks, `List<String>` deserialise could be like this: `List<String> names = mapper.readValue(jsonInput, new TypeReference<List<String>>(){});`  while get not support this way currently. So make a `String[]` to leverage current code.